src/worker.js

```diff
@@ -32,7 +39,8 @@ if (ENVIRONMENT_IS_NODE) {
     require,
     Module,
     location: {
-      href: __filename
+      // __filename is undefined in ES6 modules
+      href: typeof __filename !== 'undefined' ? __filename : undefined
```

Comment:
Is `undefined` really valid here? I think it should be `import.meta.url` instead.

Reply:
`undefined` was enough to pass the test but I have no idea how this is used or what it is :smile:  Thanks, I'll change it to that.

Out of curiosity, is it possible to test that?

Reply:
Good question, depends if anything in our JS uses `location.href` to get script's own address - presumably that's why it was being polyfilled here.

Reply:
I also wonder what are the situations where `typeof __filename` wouldn't be `undefined` in ES6 mode.